test_check_if_anagrams.py

```diff
@@ -1,6 +1,23 @@
-def are_anagrams(str1, str2):
-  # Remove the `return None` code below and add your own code.
-  return None
+def are_anagrams(string1, string2):
+  # I initially thought about the possibility of having to compare each possible iteration of the string.  That sounded ridiculous if the strings are long.
+  # Strings don't sort easily, it seems.  Could i turn the strings into lists?
+  #note: make them all the same size.  Was getting error if it is a list?
+  # I had a problem when just trying to modify the strings as is, so giving them a new object name helps
+  lowerstring1 = string1.lower()
+  lowerstring2 = string2.lower()
+  string_1 = list(lowerstring1)
+  string_2 = list(lowerstring2)
+  # sort them?
+  string_1.sort()
+  string_2.sort()
+  if string_1 == string_2:
+    return True
+  else:
+    return False
+ # This is not what I would consider pretty code, but it passes all the tests!
+ # I ran into a lot of problems.  I made two version of the code that would work correctly for all False returns
+ # I initially tried to sort them while as strings, that didn't work either! I had to look up how to split it up, and that is when I read up enough to know that lists might be the way to go.
```

Comment:
I think that pivoting to lists was a good decision! What did you try in order to sort the values as strings? If you'd like, we can talk through that as well.